glustercli/cli/gluster_version.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+
+from glustercli.cli.utils import gluster_execute
+
+class GlusterVersion:
```

Comment:
Any advantage creating class instead of function.

from glustercli.cli import glusterfs_version
print(glusterfs_version())